### PR TITLE
Document API Routes

### PR DESCRIPTION
**Pull request by ethamitc**
I've added documentation for the following API routes;

- `/news`
- `/news/{news}`

Additionally I added the `news` structure.

### CODE REVIEW

File: app/Http/Controllers/NewsController.php [hunk 1]
```diff
@@ -17,6 +20,18 @@ public function __construct()
         $this->middleware('require-scopes:public');
     }
 
+    /**
+     * Get news posts
```
Comment (nanaya) [inline app/Http/Controllers/NewsController.php:24]: title case (also for show)

**Review by nanaya — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/NewsController.php [hunk 2]
```diff
@@ -52,6 +67,17 @@ public function index()
         }
     }
 
+    /**
+     * Get news post
+     *
+     * Returns news post for the specified news slug.
+     * a
```
[the remark this answers is not in the document]
Comment (ethamitc) [reply app/Http/Controllers/NewsController.php:74]: I was about to say I'm confused too but I see it now.

File: app/Http/Controllers/NewsController.php [hunk 3]
```diff
@@ -52,6 +67,17 @@ public function index()
...
+     * ---
+     *
+     * ### Response format
+     *
+     * Renderes the news web page.
```
[the remark this answers is not in the document]
Comment (ethamitc) [reply app/Http/Controllers/NewsController.php:79]: I presume it's supposed to return JSON, when will that be fixed?

[the remark this answers is not in the document]
Comment (nanaya) [reply app/Http/Controllers/NewsController.php:79]: it's been fixed.

**Comment by peppy** [comment]
@ethamitc are you intending to apply the remaining review to this one?

**Comment by ethamitc** [comment]
> @ethamitc are you intending to apply the remaining review to this one?

@peppy Yes I am sorry, this current quad for school ends at the end of this week so I've been busy with that.

**Comment by nanaya** [comment]
Superseded by #7712.

**Pull request closed without merging**